app/grandchallenge/components/validators.py

```diff
@@ -20,3 +21,17 @@ def validate_safe_path(value):
 def validate_no_slash_at_ends(value):
     if value[0] == "/" or value[-1] == "/":
         raise ValidationError("Path must not begin or end with '/'")
+
+
+def validate_newick_tree_format(tree):
+    """Validates a Newick tree by passing it through a validator"""
+    parser = Phylo.NewickIO.Parser.from_string(tree)
+
+    try:
+        has_tree = False
+        for _ in parser.parse():
+            has_tree = True
+        if not has_tree:
+            raise ValueError("No tree found")
```

Comment:
Shouldn't this be a `ValidationError`?

Reply:
It'll eventually be a validation error because of the try-catch. I can make it a `ValidationError` from a `ValidationError`, but that seemed odd.